dtls.c

```diff
@@ -309,7 +309,7 @@ static int janus_dtls_load_keys(const char* server_pem, const char* server_key,
 /* DTLS-SRTP initialization */
 gint janus_dtls_srtp_init(const char* server_pem, const char* server_key) {
 	const char *crypto_lib = NULL;
-#if OPENSSL_VERSION_NUMBER < 0x10100000L
+#if OPENSSL_VERSION_NUMBER < 0x10100000L || defined(LIBRESSL_VERSION_NUMBER)
 	crypto_lib = "OpenSSL pre-1.1.0";
```

Comment:
Couldn't this check also be used to change the `crypto_lib` string content? This is what we use as a display value in the Janus logs, and so may be helpful to make it explicit when LibreSSL is in use.